LayoutTests/imported/w3c/web-platform-tests/mathml/relations/css-styling/padding-border-margin/padding-002-expected.txt

```diff
@@ -15,8 +15,8 @@ PASS Padding properties on mn
 PASS Padding properties on mn (rtl)
 PASS Padding properties on mo
 PASS Padding properties on mo (rtl)
-FAIL Padding properties on mover assert_approx_equals: left padding expected 30 +/- 1 but got 0
-FAIL Padding properties on mover (rtl) assert_approx_equals: left padding expected 30 +/- 1 but got 0
+PASS Padding properties on mover
+PASS Padding properties on mover
```

Comment:
Removing the `(rtl)` was not done on purpose here. Fortunately most platforms have their own expectations. Will fix in a follow-up.